examples/fcma/mvpa_classification.py

```diff
@@ -0,0 +1,139 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from sklearn import svm
+#from sklearn.linear_model import LogisticRegression
+import sys
+import os
+import logging
+from file_io import generate_epochs_info
+import numpy as np
+from scipy.stats.mstats import zscore
+import nibabel as nib
+#from sklearn.externals import joblib
+
+format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+# if want to output log to a file instead of outputting log to the console,
+# replace "stream=sys.stdout" with "filename='fcma.log'"
+logging.basicConfig(level=logging.INFO, format=format, stream=sys.stdout)
+logger = logging.getLogger(__name__)
+
+
+def prepare_data(raw_data, epoch_info):
```

Comment:
Isn't it better to make `MVPAVoxelSelector._preprocess_data` work with 2D data, make it public, and use it here instead of copying the code?